src/components/Calendar/CalendarRoot.tsx

```diff
@@ -231,6 +244,17 @@ class ScheduleCalendar extends PureComponent<ScheduleCalendarProps, ScheduleCale
             canvas.style.color = oldColor;
             calendarHeader.style.marginRight = oldMargin;
 
+            // Set text color back
+            if (isDarkMode()) {
+                for (let i = 0; i < rbc_headers.length; i++) {
+                    // delete rbc_headers[i].style;
```

Comment:
If you don't need this line anymore can you delete it instead of commenting it out?